src/index.tsx

```diff
@@ -2,10 +2,12 @@ import "@patternfly/react-core/dist/styles/base.css";
 import "@/Core/Language/Extensions";
 import React from "react";
 import { Flex } from "@patternfly/react-core";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { StoreProvider } from "easy-peasy";
 import { createRoot } from "react-dom/client";
 import { getStoreInstance } from "@/Data";
 import { Root } from "@/UI/Root";
+import AuthProvider from "./Data/Auth/AuthProvider";
```

Comment:
`import { AuthProvider } from "./Data/Auth/AuthProvider";`

(After you removed the default export)